ocs_ci/helpers/helpers.py

```diff
@@ -4480,3 +4487,103 @@ def verify_log_exist_in_pod_logs(
     )
     logger.info(f"logs osd:{pod_logs}")
     return expected_log in pod_logs
+
+
+def retrieve_cli_binary(cli_type="mcg"):
+    """
+    Download the MCG-CLI/ODF-CLI binary and store it locally.
+
+    Args:
+        cli_type (str): choose which bin file you want to download ["odf" -> odf-cli , "mcg" -> mcg-cli]
+
+    Raises:
+        AssertionError: In the case the CLI binary is not executable.
+
+    """
+    semantic_version = version.get_semantic_ocs_version_from_config()
+    if cli_type == "odf" and semantic_version < version.VERSION_4_15:
+        raise Exception(f"odf cli tool not supported on ODF {semantic_version}")
+
+    remote_path = get_architecture_path(cli_type)
+    remote_cli_basename = os.path.basename(remote_path)
+    if cli_type == "mcg":
+        local_cli_path = constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH
+    elif cli_type == "odf":
+        local_cli_path = constants.CLI_TOOL_LOCAL_PATH
+    local_cli_dir = os.path.dirname(local_cli_path)
+
+    if (
+        config.DEPLOYMENT["live_deployment"]
+        and semantic_version >= version.VERSION_4_13
+    ):
+        image = f"{constants.MCG_CLI_IMAGE}:v{semantic_version}"
+    elif semantic_version >= version.VERSION_4_15:
```

Comment:
Let's move this condition to be as first one as in case it will be 4.15 and live deployment it will be caught by previous condition ;)
```
 config.DEPLOYMENT["live_deployment"]
        and semantic_version >= version.VERSION_4_13
```

You will need to add one more condition within this for LIVE deploy and version>=4.15 to have live image from registry.redhat.com

Reply:
done